eth2near/eth2near-block-relay-rs/src/last_slot_searcher.rs

```diff
@@ -409,7 +409,8 @@ mod tests {
     const FIRST_SLOT: u64 = 1099360;
     const SLOT_WITHOUT_BLOCK: u64 = 1099364;
     const RIGHT_BOUND_IN_SLOT_SEARCH: u64 = 1099500;
-
+    const TIMEOUT: u64 = 30;
+    const TIMEOUT_STATE: u64 = 1000;
```

Comment:
And here

Reply:
fixed: https://github.com/aurora-is-near/rainbow-bridge/pull/800/commits/4aefac2a955839abc6233e6c63efe671be60feef